.github/workflows/audit.yml

```diff
@@ -38,7 +38,7 @@ jobs:
   security_audit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
```

Comment:
I verified this has matches the sha

https://github.com/actions/checkout/releases/tag/v4.2.2